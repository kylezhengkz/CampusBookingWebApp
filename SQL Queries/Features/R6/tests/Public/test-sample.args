{
    "args": [],
    "kwargs": {
        "roomName": "PR1 1",
        "minCapacity": "10",
        "maxCapacity": "200",
        "startTimeStr": "2025-06-23 17:46:05",
        "endTimeStr": "2025-06-23 20:07:05"
    }
}   