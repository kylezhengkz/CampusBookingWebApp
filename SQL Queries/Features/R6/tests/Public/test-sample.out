{'roomID': UUID('00000000-0000-0000-0000-000000000000'), 'buildingName': 'Pharmacy Residence 1', 'roomName': 'PR1 1', 'overlappingBookings': 0, 'capacity': 75, 'addressLine1': '573 Queen Blvd S', 'addressLine2': '', 'city': 'Sudbury', 'province': 'Ontario', 'country': 'Canada', 'postalCode': 'M0K 0N6'}